main.tf

```diff
@@ -50,12 +50,12 @@ locals {
     zonal    = "${var.region}"
   }
 
-  cluster_type_output_regional_zones = "${concat(google_container_cluster.primary.*.additional_zones, list(list()))}"
-  cluster_type_output_zonal_zones    = "${concat(slice(var.zones,1,length(var.zones)), list(list()))}"
+  cluster_type_output_regional_zones = "${concat(google_container_cluster.primary.*.additional_zones, list("")}"
```

Comment:
This is missing a trailing `)`, causing CI to fail

Reply:
Fixed